tests/certification/state/cockroachdb/README.md

```diff
@@ -0,0 +1,42 @@
+# CockroachDB State Store Certification
+
+The purpose of this module is to provide tests that certify the CockroachDB State Store as a stable component.
+
+## Test plan
+
+## Connection Test
+* Verify the connection is established to CockroachDB.
+    * Create component spec.
+    * Run the component with docker-compose
+    * Run dapr application with component.
+    * Ensure that you have access to the queue and connection to the queue is established.
+
+## Basic Operations Test
+* Verify that cockroachDB can save, retrieve, and delete states
+    * Save basic state information with a specified key
+    * Perform a Get Operation with that same key and ensure the the data previously written is retrieved
+    * Delete the state item
+    * Perform another Get Operation and verify that the data retreived is a nil byte array
+
+
+## ETAG Test
```

Comment:
This is already handled by Conformance tests so does not need to be in Certification tests.